src/utilities.ts

```diff
@@ -0,0 +1,28 @@
+import { Chain } from "./types";
+
+type parseTransactionResponse = Object | Buffer;
+
+/**
```

Comment:
Would be great to have an example in the jsdoc of how I could use the solana transaction vs the avalanche transaction